src/components/dapp-staking-v2/my-staking/components/CardList.vue

```diff
@@ -115,12 +115,12 @@ export default defineComponent({
       display: none;
     }
   }
-  @media (min-width: $xl) {
+  @media (min-width: $lg) {
```

Comment:
@sirius651 @Kahonnohak This breaks UI here
![image](https://user-images.githubusercontent.com/92044428/197764548-4b30e266-c204-4d37-b7d9-7e4fc7dede6d.png)

Reply:
it was expected result for me. give me feedback for this @Kahonnohak

Reply:
But it breaks UI. We should fix it. @sirius651 @Kahonnohak 

1076px~1500px
![image](https://user-images.githubusercontent.com/92044428/197919264-c9c414e7-0ea3-43b0-9f77-668ec7dc1e59.png)

Reply:
I've modified in [this](https://github.com/AstarNetwork/astar-apps/pull/553) PR. Please take a look and compare the staging URL. @Kahonnohak @sirius651

Reply:
So this is what I see now and I thought Roy wanted to fix this not to happen? I personally think this is not a critical issue and we can leave it for the next weeks. But if you have thought this has been fixed I didn't see it yet. Of course, it would be great if it can behave as it is in the design soonish.

<img width="1098" alt="Screenshot 2022-10-26 at 08 14 42" src="https://user-images.githubusercontent.com/77480847/197960036-6982efcc-9871-45f9-adf9-71097ad43f61.png">
<img width="1076" alt="Screenshot 2022-10-26 at 08 15 03" src="https://user-images.githubusercontent.com/77480847/197960044-f015cbc4-e008-4077-927b-b60345037106.png">

Reply:
> So this is what I see now and I thought Roy wanted to fix this not to happen? I personally think this is not a critical issue and we can leave it for the next weeks. But if you have thought this has been fixed I didn't see it yet. Of course, it would be great if it can behave as it is in the design soonish.

I've fixed the issue at [this](https://github.com/AstarNetwork/astar-apps/pull/553) PR. Please take a look, I don't want to publish the broken UI...